libs/application/templates/inheritance-report/src/fields/CalculateTotalAssets/index.tsx

```diff
@@ -0,0 +1,42 @@
+import { getValueViaPath } from '@island.is/application/core'
+import { FieldBaseProps } from '@island.is/application/types'
+import { formatCurrency } from '@island.is/application/ui-components'
+import { Box, Text } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { FC, useEffect, useState } from 'react'
+import { useFormContext } from 'react-hook-form'
+import { m } from '../../lib/messages'
+
+export const CalculateTotalAssets: FC<FieldBaseProps> = ({ application }) => {
+  const { answers } = application
+  const { formatMessage } = useLocale()
+  const { setValue } = useFormContext()
+
+  const [total] = useState(
+    (getValueViaPath(answers, 'assets.otherAssets.total') as number) +
+      (getValueViaPath(answers, 'assets.money.total') as number) +
+      (getValueViaPath(answers, 'assets.stocks.total') as number) +
+      (getValueViaPath(answers, 'assets.claims.total') as number) +
+      (getValueViaPath(answers, 'assets.bankAccounts.total') as number) +
+      (getValueViaPath(answers, 'assets.inventory.total') as number) +
+      (getValueViaPath(answers, 'assets.vehicles.total') as number) +
+      (getValueViaPath(answers, 'assets.realEstate.total') as number),
+  )
```

Comment:
If one of these is undefined the result will be `NaN`. Is that okay?
Is all of this required before reaching this state?